processor/compile/image.ts

```diff
@@ -1,17 +1,17 @@
 import type { Image } from 'mdast';
 
 const image = (node: Image) => {
-  const { align, className, width } = node.data?.hProperties || {};
-  const complexImage: boolean = Boolean(width) || Boolean(className) || Boolean(align);
-
+  const { align, border, width, src } = node.data?.hProperties || {};
+  const complexImage: boolean = Boolean(width) || Boolean(border) || Boolean(align);
+  
   if (complexImage) {
     const attributes = Object.keys(node.data?.hProperties)
       .map(key => `${key}="${node.data?.hProperties[key]}"`)
       .join(' ');
     return `<Image ${attributes} />`;
   }
 
-  return `![${node.alt}](${node.url}${node.title ? ` "${node.title}")` : ')'}`;
-};
+  return `![${node.alt ?? ''}](${src ? src : node.url}${node.title ? ` "${node.title}")` : ')'})`;
```

Comment:
I think there's an extra `)`!

Reply:
hah! there's always an extra `)` or `'` whenever i edit this line 😂